sdk/core/azure-core/src/main/java/com/azure/core/http/rest/StreamResponse.java

```diff
@@ -22,9 +30,22 @@ public final class StreamResponse extends SimpleResponse<Flux<ByteBuffer>> imple
      * @param statusCode The status code of the HTTP response.
      * @param headers The headers of the HTTP response.
      * @param value The content of the HTTP response.
+     * @deprecated Use {@link #StreamResponse(HttpResponse)}
      */
+    @Deprecated
```

Comment:
RestProxy will likely be calling this constructor, we'll want to update it to use the new constructor.

Reply:
Yup I did that and I added couple of tests for this https://github.com/kasobol-msft/azure-sdk-for-java/blob/72c2f00cde25d83b9ce9c3ed598622f1ed1c8475/sdk/core/azure-core/src/test/java/com/azure/core/http/rest/RestProxyTests.java#L107-L108 .